### PR TITLE
Script to use docker image to analyze demo repos

### PR DESCRIPTION
**Pull request by yusefsaraby**

### CODE REVIEW

**Review by perrito666 — COMMENTED** [review]
Great Start!!!

I added some comments, mostly suggestions and nits, feel free to disregard them, or ping me for discussion.

Let me know if you decide to take or ignore these (and suggest alternatives) and i'll do another pass.

Also, remember to pass this through `shellcheck` before we merge it just in case we are missing something.

File: cli-examples/cli-analyze-demo-applicaitons.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+run_demo_java() {
+    echo "Run Java Demo: Parameters passed: $1"
+    shiftleft_analyze_code "java" "$1"
+    docker run --rm -e SHIFTLEFT_ACCESS_TOKEN -v $1:/myvol -it $DOCKER_IMAGE_NAME /bin/bash -c "cd /myvol; mvn clean package; sl analyze --wait --sca --app shiftleft-java-demo --cpg --java /myvol/target/hello-shiftleft-0.0.1.jar"
+}
+
+# $1: absolute path to code or binaries
+run_demo_python() {
+    echo "Run Python Demo: Parameters passed: $1"
+    echo "Python path is: " pwd
```
[the remark this answers is not in the document]
Comment (yusefsaraby) [reply cli-examples/cli-analyze-demo-applicaitons.sh:45]: removed it, it was for debugging purposes

**Comment by yusefsaraby** [comment]
@perrito666 I applied the PR comments. Please review again

File: cli-examples/cli-analyze-demo-applicaitons.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+run_demo_java "$DOCKER_WORK_DIR/shiftleft-java-demo" || true
+
+echo "************** PYTHON *************"
+git clone https://github.com/ShiftLeftSecurity/shiftleft-python-demo.git || true
+run_demo_python "$DOCKER_WORK_DIR/shiftleft-python-demo" || true
+
+echo "************** JS *************"
+git clone https://github.com/ShiftLeftSecurity/shiftleft-js-demo.git || true
+shiftleft_analyze_code "js" "$DOCKER_WORK_DIR/shiftleft-js-demo" || true
+
```
[the remark this answers is not in the document]
Comment (perrito666) [reply cli-examples/cli-analyze-demo-applicaitons.sh:67]: @yusefsaraby what about this one?

[the remark this answers is not in the document]
Comment (yusefsaraby) [reply cli-examples/cli-analyze-demo-applicaitons.sh]: done

**Review by perrito666 — APPROVED** [review]

**Pull request merged**